alpha/lib/model/UserLoginDataPeer.php

```diff
@@ -805,7 +805,24 @@ protected static function isExcludedAdminRole($partner, $userRoleNames)
 	{
 		return in_array($partner->getExcludedAdminRoleName(), explode(',', $userRoleNames), true);
 	}
-	
+
+	protected static function getExcludedAdminRoleUsersNumber($partner)
+	{
+		$excludedRoleName = $partner->getExcludedAdminRoleName();
+		if (!$excludedRoleName || $excludedRoleName == '')
+		{
+			return 0;
+		}
+		$excludedUserRole = UserRolePeer::getByNameAndPartnerId($excludedRoleName, $partner->getId());
+		if (!$excludedUserRole)
+		{
+			return 0;
+		}
+		$c = new Criteria();
+		$c->add(KuserToUserRolePeer::USER_ROLE_ID, $excludedUserRole->getId());
```

Comment:
try to check 
$excludedUserRole->countKuserToUserRoles();
instead, it seems to do what you need